packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+
+import DebitCard from './template'
+
+class DebitCardContainer extends React.PureComponent<Props> {
+  render() {
+    return <DebitCard />
+  }
+}
+
+const connector = connect()
```

Comment:
Why do you need redux here?

Reply:
I will use it in the following pr after adding the async calls for obtaining cards info

Reply:
I removed anyway, will add all in next pr